src/main/java/dansplugins/factionsystem/commands/DisbandCommand.java

```diff
@@ -57,13 +62,18 @@ public void execute(CommandSender sender, String[] args, String key) {
         if (args.length == 0) {
             if (!checkPermissions(sender, "mf.disband")) return;
             if (!(sender instanceof Player)) { // ONLY Players can be in a Faction
-                sender.sendMessage(translate(getText("OnlyPlayersCanUseCommand")));
+                if (!new MedievalFactions().USE_NEW_LANGUAGE_FILE) {
```

Comment:
Pass in the plugin instance rather than constructing a new one

Reply:
Seconded.

Reply:
Done.